docs/dev/data/0.csv

```diff
@@ -0,0 +1,2910 @@
+26.0,4.0,5.0,8.0,-1.0,-13.0,-109.0,-66.0,-9.0,2.0,4.0,13.0,-18.0,-30.0,-119.0,-45.0,17.0,6.0,-1.0,-9.0,27.0,20.0,91.0,71.0,-26.0,-1.0,0.0,1.0,13.0,20.0,-62.0,-19.0,29.0,2.0,2.0,0.0,-23.0,-1.0,-80.0,4.0,-7.0,-6.0,-12.0,-27.0,-11.0,-16.0,-67.0,-8.0,-27.0,1.0,1.0,13.0,-8.0,-11.0,21.0,-28.0,61.0,4.0,8.0,5.0,4.0,-7.0,-59.0,16.0,0
```

Comment:
Do you think we should pull these down as temp files?  They bloat your repo and all you really need is some examples..

Maybe just `wget` from kaggle as part of a script and ignore these files?

Reply:
I don't think this is a trivial process... Kaggle make it very difficult.

Reply:
Okay